modules/plottinggl/include/modules/plottinggl/processors/parallelcoordinates/pcpaxissettings.h

```diff
@@ -35,6 +35,7 @@
 #include <inviwo/core/properties/boolproperty.h>
 #include <modules/opengl/texture/texture2d.h>
 
+#include <inviwo/dataframe/properties/colormapproperty.h>
```

Comment:
used in here?

Reply:
Good catch, not since refactoring. Removed it